### PR TITLE
Add job failed trigger

### PR DESCRIPTION
**Pull request by arikalon1**
Add report job failure action
Add job events enrichment action

### CODE REVIEW

File: docs/catalog/triggers/smart.rst [hunk 1]
```diff
@@ -38,6 +38,23 @@ Trigger parameters:
 * ``rate_limit``: Limit firing to once every `rate_limit` seconds
 
 
+.. _on_job_failure:
+
+* ``on_job_failure``
+
+This trigger will fire when a Job becomes failed.
```
[the remark this answers is not in the document]
Comment (arikalon1) [reply docs/catalog/triggers/smart.rst:45]: fixed

File: playbooks/robusta_playbooks/common_actions.py [hunk 2]
```diff
@@ -31,3 +31,49 @@ def customise_finding(event: ExecutionBaseEvent, params: FindingOverrides):
...
+
+
+@action
+def create_finding(event: ExecutionBaseEvent, params: FindingFields):
+    """
+    Create a new finding.
+
+    All messages from Robusta are represented as a Finding object.
+
+    This action lets create a Finding that Robusta sends.
```
Comment (Sheeproid) [inline playbooks/robusta_playbooks/common_actions.py:61]: "This action lets create a Finding that Robusta sends." English errors

File: playbooks/robusta_playbooks/common_actions.py [hunk 3]
```diff
@@ -31,3 +31,49 @@ def customise_finding(event: ExecutionBaseEvent, params: FindingOverrides):
...
+
+    """
+    subject = event.get_subject()
+    labels = defaultdict(lambda: "<missing>")
+    labels.update({
+        "name": subject.name,
+        "kind": subject.subject_type,
+        "namespace": subject.namespace if subject.namespace else "<missing>",
+        "node": subject.node if subject.node else "<missing>",
```
Comment (Sheeproid) [inline playbooks/robusta_playbooks/common_actions.py:70]: does the "<missing>" makes sense? why not an empty label? for aggregation in the UI?

**Review by Sheeproid — APPROVED** [review]

File: playbooks/robusta_playbooks/common_actions.py (hunk 2 above)
Comment (arikalon1) [reply playbooks/robusta_playbooks/common_actions.py:61]: fixed

**Pull request merged**